packages/backend-for-frontend/src/services/authorizationService.ts

```diff
@@ -167,10 +171,16 @@ export function authorizationServiceBuilder(
         Authorization: `Bearer ${serialized}`,
       };
 
-      const tenantBySelfcareId =
-        await tenantProcessClient.selfcare.getTenantBySelfcareId({
+      const tenantBySelfcareId = await tenantProcessClient.selfcare
+        .getTenantBySelfcareId({
           params: { selfcareId },
           headers: internalHeaders,
+        })
+        .catch((err) => {
+          throw isAxiosError(err) &&
+            err.response?.status === HTTP_STATUS_NOT_FOUND
+            ? tenantBySelfcareIdNotFound(selfcareId)
```

Comment:
@galales / @Viktor-K  A question: if it's for security reasons, shouldn't we return a more generic problem?
In this way, even if the code is mapped to 403, the problem we are returning still says that the tenant cannot be found, so it's theoretically possible to infer if a tenant exists or not by brute forcing selfcareId and seeing if the returned error is this one or another. I would just throw a more generic `unauthorizedError` that already exists in the common errors. 
Maybe it's impossible or it's not a problem - in this case, ignore this comment.

Reply:
@galales the [previous comment](https://github.com/pagopa/interop-be-monorepo/pull/1720#discussion_r2024467759) require text message changes, can be consider it always valid for it and others endpoints? 
Anyway I suppose that security assessment was done to cover Authorization API in BFF and this messages and status HTTP code returned were already reviewed.